src/ol/PluggableMap.js

```diff
@@ -1168,6 +1153,27 @@ class PluggableMap extends BaseObject {
         this.renderer_ = this.createRenderer();
       }
 
+      this.mapBrowserEventHandler_ = new MapBrowserEventHandler(
+        this,
+        this.moveTolerance_
+      );
+      for (const key in MapBrowserEventType) {
+        this.mapBrowserEventHandler_.addEventListener(
```

Comment:
Will these get removed by `this.mapBrowserEventHandler_.dispose();`? Otherwise it might cause leaks?

Reply:
Yes, these will get removed.

Reply:
Mind to explain me how? just interested :)